core/storage/blog/gae_models_test.py

```diff
@@ -235,6 +281,33 @@ def setUp(self):
         self.blog_post_rights_draft_model.update_timestamps()
         self.blog_post_rights_draft_model.put()
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = {
+            'editor_ids': base_models.EXPORT_POLICY.EXPORTED,
+            'blog_post_is_published': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE
+        }
+        self.assertEqual(
+            blog_models.BlogPostRightsModel.get_export_policy(),
+            sample_dict)
+
+    def test_get_field_name_mapping_to_takeout_keys(self) -> None:
+        self.assertEqual(
+            blog_models.BlogPostRightsModel.
+            get_field_name_mapping_to_takeout_keys(),
+            {
+                'editor_ids': 'editable_blog_post_ids'
+            })
+
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            blog_models.BlogPostRightsModel.
+                get_model_association_to_user(),
+            base_models.
+            MODEL_ASSOCIATION_TO_USER.ONE_INSTANCE_SHARED_ACROSS_USERS)
```

Comment:
Could you indent this line to make this easier to read?

Reply:
Done!